setup.ps1

```diff
@@ -11,6 +11,10 @@ Param(
     [Parameter(mandatory=$true)]
     $ServicePrincipalName,
 
+    [string]
```

Comment:
@vyta it is because we use it to authenticate the gh actions and also use the same SP for the csi driver via the az iot ops init command than creating a new random SP by the command installation itself. 

Btw, I didn't change this...was already there last winter

Reply:
This is referring to $HciAksGroupName variable not the service principle

Reply:
Ok, then my answer is "NO" but if I make it optional or provide another setup-HCI.ps1 file or instruction to create a group then people will get confused IMHO...or what is your suggestion here?

Reply:
I think make it optional and then provide steps in the readme, I'll add my suggestion to the readme file in a comment on this PR

Reply:
Opened issue for this: #17